freshmaker/pyxis_gql.py

```diff
@@ -496,3 +549,59 @@ def find_images_by_names(self, names):
             page_num += 1
 
         return images
+
+    def find_images_by_repository(self, repository: str, auto_rebuild_tags: List[str] = []) -> List:
```

Comment:
Mutable default arguments should be avoided in Python: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments